lib/src/null_span.dart

```diff
@@ -12,6 +12,7 @@ import 'package:source_span/source_span.dart';
 class NullSpan extends SourceSpanMixin {
   @override
   final SourceLocation start;
+
```

Comment:
nit: Not sure why you added a line between these members but not between others.

Reply:
`dartfmt`.

Reply:
Hrmmm.  Actually, I have no idea.  Restored.